File: 1009-complement-of-base-10-integer/1009-complement-of-base-10-integer.py
class Solution:
    def bitwiseComplement(self, n: int) -> int:
        if n==0:
            return 1
        i=0
        num=n
        while(i<31):
            val=1<<i
            if(val>n):
                break
            num^=val
            i+=1
        return num

File: 1009-complement-of-base-10-integer/1009-complement-of-base-10-integer.java
class Solution {
    public int bitwiseComplement(int n) {
        String str=Integer.toBinaryString(n);
        String s="";
        for(int i=0; i<str.length(); i++){
            if(str.charAt(i)=='1')s+='0';
            else s+='1';
        }
        int num=Integer.parseInt(s,2);
        return num;
    }
}